### PR TITLE
Refactor categories

### PR DESCRIPTION
**Pull request by kennedy**

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -25,6 +25,7 @@
                     <ul class="icons">
                     </ul>
                 </header>
+                <!-- Sidebar -->
```
Comment (kennedy) [inline _layouts/default.html:28]: Comment for visual reference when Sidebar and Content starts

File: _layouts/page.html [hunk 2]
```diff
@@ -1,39 +1,10 @@
...
+---
+layout: default
+---
+<!-- Post -->
+<section>
+    <header class="main">
+        <h1>{{ page.title }}</h1>
+    </header>
+    {{ content }}
+</section>
```
Comment (kennedy) [inline _layouts/page.html:10]: Following the same structure as`index.html`

File: all-recipes.md [hunk 3]
```diff
@@ -2,8 +2,9 @@
 layout: page
 title: All Recipes
 ---
+{% assign sorted =  site.recipes | sort_natural: "title" %}
 <ul>
-{% for r in site.recipes %}
+{% for r in sorted %}
```
Comment (kennedy) [inline all-recipes.md:7]: Its prob not really important, but insuring the recipes are sorted by title, just like the sidebar.

File: categories.md [hunk 4]
```diff
@@ -1,14 +1,15 @@
 ---
 layout: page
-title: Tags
+title: Categories
 ---
-{% assign tags =  site.recipes | map: 'tags' | join: ','  | split: ',' | uniq %}
+{% assign tags =  site.recipes | map: 'tags' | uniq | sort_natural %}
```
Comment (kennedy) [inline categories.md:5]: There doesnt seem to be a need for joining and splitting by `,`. uniq does that natively

File: categories.md [hunk 5]
```diff
@@ -1,14 +1,15 @@
...
+{% assign sorted =  site.recipes | sort_natural: "title" %}
 {% for tag in tags %}
-  <h3 id="{{tag | slugify }}">{{ tag | slugify }}</h3>
+  <h3 id="{{tag | slugify }}">{{ tag | upcase}}</h3>
```
Comment (kennedy) [inline categories.md:8]: I think it looks better upper case. not a huge deal, if it looks off i can change it back

**Review by kennedy — COMMENTED** [review]
This is converting a very important layout, the page.html. This page is used for every recipe, the all-recipe, and every other general page across the site.

The changes would make creating a new page much easier.

**Comment by thenaterhood** [comment]
Pssst hey. Still working on this? It's still marked as a draft :)

**Comment by kennedy** [comment]
> Pssst hey. Still working on this? It's still marked as a draft :)

whooooppppssss

**Pull request merged**